package/build-package.sh

```diff
@@ -3,32 +3,45 @@
 # This script builds a .deb package from the binaries in the .stack-work
 # directory. It is intended to be run on a Debian-like system.
 #
-#   Usage: VERSION=0.0.0 ./build-package.sh
+#   Usage: VERSION=0.0.0 fakeroot ./build-package.sh
 
 # Fail early of any of the commands below fail.
 set -e
```

Comment:
For this script, you probably want to change `#!/bin/sh` to `#!/bin/bash` and:

```
set -eo pipefail
```

Without `pipefail`, pipelines have the exit status of the last command in the pipeline.

So `cat nonexistentfile | envsubst > "$PKGNAME/DEBIAN/control"` would not make the script crash.

Reply:
Done.